### PR TITLE
[Calendar] add action support to manifest

### PR DESCRIPTION
**Pull request by xhr0804**
### Purpose
*What is the context of this pull request? Why is it being done?*
First step of #3074 
### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*
Add main top-level intent as action in manifest 
### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

**Comment by xieofxie** [comment]
I am not sure if we should include token in the input, ask in email then

**Review by lzc850612 — APPROVED** [review]

File: skills/csharp/calendarskill/wwwroot/manifest/manifest-1.0.json [hunk 1]
```diff
@@ -23,9 +23,242 @@
...
   "activities": {
+    "createEvent": {
+      "description": "Create a new event",
+      "type": "event",
+      "name": "CreateEvent",
+      "value": {
+        "$ref": "#/definitions/eventInfo"
+      },
+      "resultValue": {
+        "$ref": "#/definitions/operationStatus"
```
Comment (xieofxie) [inline skills/csharp/calendarskill/wwwroot/manifest/manifest-1.0.json:34]: One more thought: if the user doesn't fill all slots in PVA, the caller could not know the result. 
Even if all slots are filled, they are not guaranteed to be mapped to event result directly.
So maybe we should return the created event info in resultValue. Same applies for similar actions.

File: skills/csharp/calendarskill/wwwroot/manifest/manifest-1.0.json [hunk 2]
```diff
@@ -23,9 +23,242 @@
...
+        "$ref": "#/definitions/dateInfo"
+      },
+      "resultValue": {
+        "$ref": "#/definitions/eventSummary"
+      }
+    },
     "message": {
       "type": "message",
-      "description": "Receives the users utterance and attempts to resolve it using the skill's LU models"
+      "description": "Receives the user's' utterance and attempts to resolve it using the skill's LU models"
```
Comment (xieofxie) [inline skills/csharp/calendarskill/wwwroot/manifest/manifest-1.0.json]: user's utterance

Comment (xhr0804) [reply skills/csharp/calendarskill/wwwroot/manifest/manifest-1.0.json]: fixed

File: skills/csharp/calendarskill/wwwroot/manifest/manifest-1.0.json (hunk 1 above)
Comment (darrenj) [reply skills/csharp/calendarskill/wwwroot/manifest/manifest-1.0.json:34]: If the user doesn't provide all the slots then the Skill should prompt as usual for anything missing (if required)

**Review by darrenj — APPROVED** [review]

**Pull request merged**